smslibrary/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,21 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.eis.smslibrary" />
+    package="com.eis.smslibrary">
+
+    <uses-permission android:name="android.permission.SEND_SMS" />
+    <uses-permission android:name="android.permission.RECEIVE_SMS" />
+
+    <application>
+        <receiver
+            android:name=".SMSReceivedBroadcastReceiver"
+            android:enabled="true"
+            android:exported="true"
+
+            android:permission="android.permission.BROADCAST_SMS">
+            <intent-filter
+                android:priority="999">
```

Comment:
Is this priority specification necessary?

Reply:
No, but it could be useful if you need to execute the broadcast receiver code before the other receivers